src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -948,6 +950,15 @@ module Types = struct
               ~resolve:(fun {ctx= coda; _} key ->
                 AccountObj.get_best_ledger_account coda key ) ] )
 
+    let register_hardware_wallet :
+        (Coda_lib.t, Coda_numbers.Hardware_wallet_nonce.t option) typ =
+      obj "RegisterHardwareWallet" ~fields:(fun _ ->
```

Comment:
It seems like we could use the AddAccountPayload for this, maybe. If not, we can probably use the `<X>Payload` naming convention here